src/api.js

```diff
@@ -5,7 +5,13 @@ var API = (function() {
     address: '21275 Lakeshore Dr\nSainte-Anne-de-Bellevue\nQuébec\nH9X 3L9',
     maritalStatus: 'Widowed',
     children: '8',
-    income: '$10,000',
+    income: {
+      employerName: 'Acme Corp',
```

Comment:
Since we're adding a company name now, let's add this to the intro paragraph on the main dashboard page.

(Also, v sad to lose BLORB CORP 😢 )